.github/notebook_lists/vanilla_notebooks.txt

```diff
@@ -3,3 +3,4 @@ recipes/Guard-Rails/HAP.ipynb
 recipes/Auto_Documentation/Auto_Documentation.ipynb
 recipes/Text_to_Shell_Exec/Text_to_Shell_Exec.ipynb
 recipes/Text_to_SQL/Text_to_SQL.ipynb
+recipes/generate_sql_and_execute/generate_sql_and_exec.ipynb
```

Comment:
Recipes in this file are tested during the CI build. I am not sure this recipe can be tested since it seems to include a number of manual steps instead of executable cells. If that is the case, then it should not be listed in this file.

It should however be mentioned in the readme like other recipes are.